documentation/README.md

```diff
@@ -185,7 +185,7 @@ When you **right-click on a block**, the following options are available:
 
 ![Duplicate](./duplicate.png "Duplicate")
 
-**Help**: The Help option displays a detailed help screen for the selected block. This screen provides valuable information about the block’s functionality, purpose, and usage instructions, making it easier for users to understand and effectively incorporate the block into their projects.
+**Help**: The Help option displays a detailed help screen for the selected block. This screen provides valuable information about the block’s functionality, purpose, and usage instructions, making it easier for users to understand and effectively incorporate the block into their projects. By using the right-click context menu in Music Blocks, you can quickly perform common tasks and manipulate blocks on the workspace. This can help you to work more efficiently and effectively in your projects. Additionally, help screens allow you to insert example blocks directly into your program by clicking on the download icon *(labeled Load this block)*.
```

Comment:
This can help you work more efficiently and effectively in building your projects. Additionally, the help screen allows you to insert blocks from the help examples directly into your program by clicking on the download icon *(labeled "Load this Block")*.

Reply:
or maybe `(labeled "Load this Block" in the image below)`.

Also, I don't think we need italics there, especially not for "labeled".